README.md

```diff
@@ -73,20 +75,27 @@ npm install --save capnp-ts
 You may want the schema compiler as well (can also be installed locally):
 
 ```shell
-npm install -g capnpc-ts
+npm install -g capnpc-ts # For TypeScript
+# OR
+npm install -g capnpc-js # For JavaScript
```

Comment:
"Registering" is simply a matter of publishing first.

Suppose I'll just do that today with zero fanfare so nobody tries using it yet...